src/components/calcite-tip/calcite-tip.scss

```diff
@@ -68,7 +68,7 @@
 }
 
 .image-frame {
-  width: theme("width.[1/4");
```

Comment:
This one was missing a closing bracket. I'm not sure it was failing but maybe we can use some linting here.